components/builder-depot/src/server.rs

```diff
@@ -792,13 +806,14 @@ fn upload_package(req: &mut Request) -> IronResult<Response> {
         return Ok(Response::with(status::NotImplemented));
     };
 
+    let mut visibilities = Vec::new();
+    visibilities.push(OriginPackageVisibility::Public);
+    visibilities.push(OriginPackageVisibility::Hidden);
+    visibilities.push(OriginPackageVisibility::Private);
+
```

Comment:
There's a function that will give you this vector in helpers.

https://github.com/habitat-sh/builder/blob/master/components/builder-http-gateway/src/http/helpers.rs#L185-L191

Reply:
cool.. thanks!